a_locs_poly_setup.R

```diff
@@ -9,8 +9,10 @@ tar_source("a_locs_poly_setup/src/")
 # NW = Northern Water.
 
 # create folder structure
-dir.create("a_locs_poly_setup/nhd/")
-dir.create("a_locs_poly_setup/out/")
+suppressWarnings({
+  dir.create("a_locs_poly_setup/nhd/")
```

Comment:
nice built-in directory folder creation :)

Reply:
thank you!! without the supress warnings it would give me warnings in the targets pipeline, so that was a late add so they didn't show up on rendered docs/targets runs.